tests/linux_kernel/kmod/drgn_test.c

```diff
@@ -847,6 +848,41 @@ static void drgn_test_idr_exit(void)
 	idr_destroy(&drgn_test_idr_sparse);
 }
 
+// wait-queue
+static struct task_struct *drgn_test_waitq_kthread;
+static wait_queue_head_t drgn_test_waitq;
+static wait_queue_head_t drgn_test_empty_waitq;
+static bool exit_waitq_test;
+
+static int drgn_test_waitq_kthread_fn(void *arg)
+{
+	wait_event_interruptible(drgn_test_waitq, exit_waitq_test);
```

Comment:
I think we can get rid of `exit_waitq_test` and use `kthread_should_stop()` directly.
```suggestion
static int drgn_test_waitq_kthread_fn(void *arg)
{
	wait_event_interruptible(drgn_test_waitq, kthread_should_stop());
```

Reply:
Good point. done now.